modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,156 @@
+* Before this update, OpenShift console UI would use a hardcoded pipeline run template as a fallback template when a runtime template was not found in the OpenShift namespace. This update provides a default pipeline run template for the console to use, `pipelines-as-code-template-default`, in the `pipelines-as-code` config map. 
+* With this update, {pac} supports Tekton Pipelines 0.44.0 minimal status. 
+* With this update, {pac} supports Tekton v1 API, which means {pac} is now compatible with Tekton v0.44 and later.
+* With this update, you can configure custom console dashboards in addition to configuring a console for OpenShift and Tekton dashboards for k8s.
+* With this update, {pac} detects the installation of a GitHub application initiated using the `tkn pac create repo` command and does not require a GitHub webhook if it was installed globally.
+* Before this update, if there was an error on a `PipelineRun` execution and not on the tasks attached to `PipelineRun`, {pac} would not report the failure properly. With this update, {pac} reports the error properly on the GitHub checks when a `PipelineRun` could not get created.  
+* With this update, {pac} includes a `target_namespace` variable, which expands to the currently running namespace where the `PipelineRun` is executed.
+* With this update, {pac} lets you bypass GitHub enterprise questions in the CLI bootstrap GitHub application.
+* With this update, {pac} does not report errors when the repository CR was not found.
+* With this update, {pac} reports an error if multiple pipeline runs with the same name were found.
+
+
+[id="breaking-changes-1-10_{context}"]
+== Breaking changes
+
+* This update removes support for cluster and `CloudEvent` pipeline resources from Tekton CLI. You cannot create pipeline resources by using the `tkn pipelineresource create` command. Also, the pipeline resources are not supported anymore in the `start` command of a task, cluster task, and pipeline.
+* This update removes `tekton` as a provenance format from Tekton Chains.
+
+
+[id="deprecated-features-1-10_{context}"]
+== Deprecated and removed features
+
+* In {pipelines-title} 1.10, the `ClusterTask` commands are now deprecated and are planned to be removed in a future release. The `tkn task create` command is also deprecated with this update. 
+* In {pipelines-title} 1.10, the flags `-i` and `-o` that were used with the `tkn task start` command are now deprecated because the v1 API does not support pipeline resources.
+* In {pipelines-title} 1.10, the flag `-r` that was used with the `tkn pipeline start` command is deprecated because the v1 API does not support pipeline resources.
+* In {pipelines-title} 1.10, the pipeline default embedded status will be moved to minimal in a future release. This update sets the `openshiftDefaultEmbeddedStatus` parameter to `both` with full and embedded status. Also, the flag to change the default embedded status will be removed in a future release.
```

Comment:
1.10, This update sets the `openshiftDefaultEmbeddedStatus` parameter to `both` with `full` and `minimal` embedded status. Also, the flag to change the default embedded status will be removed and the pipeline default embedded status will be moved to minimal in a future release.

Reply:
Incorporated